.travis.yml

```diff
@@ -7,6 +7,23 @@ matrix:
   include:
   - jdk: oraclejdk7
     script: mvn -q deploy --settings .travis-settings.xml -Dno.gem.deploy=true
+  - jdk: oraclejdk7
+    env: ANDROID_EXAMPLES=true
+    language: android
+    android:
```

Comment:
any particular reason why the steps are not in chronological order (install, before_script, script)?